src/App.js

```diff
@@ -0,0 +1,200 @@
+import React, { useState } from "react";
+import './App.css';
+import Valgboks from "./components/Valgboks/Valgboks";
+import Moment from "moment";
+import Header from "./components/Header/Header";
+import Liste from "./components/Liste/Liste";
+import Kalender from "./components/Kalender/Kalender";
+import { 
+  BrowserRouter as Router,
+  Route,
+  Switch } from "react-router-dom";
+import Snow from "./components/Snow/Snow"
+import NavLinks from "./components/NavLinks/NavLinks"
+
+
+const App = () => {
+
+  const [tasks, setTasks] = useState([]);
+  const [isTasksGaver, setisTasksGaver] = useState(false)
+  const [isTasksPraktisk, setisTasksPraktisk] = useState(false)
+  const [snow, setSnow] = useState(false);
+  
+
+  //---------------------------------------------------------------------
+  //LEGGE TIL TASKS
+  //---------------------------------------------------------------------  
+
+  //Oppretter nye objekter i tasks
+  class Task {
+    constructor(oppgave, ansvar, frist, className) {
+      this.oppgave = oppgave;
+      this.ansvar = ansvar;
+      this.frist = Moment(frist).format("DD/MM");
+      this.className = className;
+      this.id = "key" + Math.floor(Math.random()*1000);
+    }
+  }
+
+  //Oppdaterer tasks
+  const handleAddTask = () => {
+    let COPY = [...tasks];
+
+    let oppdragsInput = document.getElementById("oppdrag_inp");
+    let ansvarsInput = document.getElementById("ansvar_inp");
+    let fristInput = document.getElementById("frist_inp");
+    let gave_radio = document.getElementById("gave_radio");
+    let praktisk_radio = document.getElementById("praktisk_radio");
+    let className = "";
+
+    //Utdeler klassenavn til ny oppgave basert på om bruker vil legge det til i "Gaver" eller "Praktisk". 
+    //Endrer state for isTasks
+    if (gave_radio.checked) {
+      className="gaveliste"
+      setisTasksGaver(true)
+    } else if (praktisk_radio.checked) {
+      className="praktiskliste"
```

Comment:
Literalsene "praktiskliste" og "gaveliste" bør inn i noen konstanter. Når du skriver dem ofte, er risikoen for skrivefeil ganske stor. Du sparer deg for en del risiko ved å legge dem inn i variabler.